compute/instances.go

```diff
@@ -287,7 +287,7 @@ type CreateInstanceInput struct {
 	LocalityStrict  bool
 	LocalityNear    []string
 	LocalityFar     []string
-	Metadata        map[string]string
+	Metadata        map[string]interface{}
 	Tags            map[string]string //
```

Comment:
Should this Tags input be allowed to take `interface{}` too?

Reply:
VMAPI doesn't disallow it, so I'd say it should, yeah. I'll update.